bitarray.go

```diff
@@ -265,3 +265,23 @@ func (bits *BitArray) ToArray() []int {
 
 	return ints
 }
+
+func (bits *BitArray) ToByteArray() []byte {
+	return bits.bytes
+}
+
+// New create a new BitArray with length(bits).
+func FromByteArray(length int, data []byte) (*BitArray, error) {
+	if data == nil {
+		return nil, errors.New("data can't be nil")
+	}
+	lenpad := nwords(length) * _BytesPW
+	if len(data) != lenpad {
```

Comment:
I think we should create a new bytes slice with `lenpad` then copy content from the origin slice, also the `length` should be greater or equal to `len(data)`.